src/pe/mod.rs

```diff
@@ -42,7 +42,7 @@ pub struct PE<'a> {
     /// Data for any imported symbols, and from which `dll`, etc., in this binary
     pub import_data: Option<import::ImportData<'a>>,
     /// The list of exported symbols in this binary, contains synthetic information for easier analysis
-    pub exports: Vec<export::Export<'a>>,
+    pub exports: Option<Vec<export::Export<'a>>>,
```

Comment:
I typically prefer an empty vector for zero exports, as its less heavy-weight on the user, and in principle, they still need to check whether the vector is empty anyway.

In other words, in this usecase, making it optional doesn't really give us any extra semantic help here, so I'd just leave it as a regular vector (and empty if no exports), unless there's some other compelling reason I'm not seeing